packages/grafana-ui/src/types/datasource.ts

```diff
@@ -370,7 +370,7 @@ export interface DataQueryError {
 }
 
 export interface ScopedVar {
-  text: any;
+  text?: any;
```

Comment:
@torkelo is this change safe?

Reply:
ight, I feel like introducing optional here is not neccessary. Main driver here is table/renderer.ts / renderRowVariables. I suggest using stringified value there as `text` property value